src/Behat/Cores/Drupal8.php

```diff
@@ -230,9 +230,16 @@ public function getLastEntityId($entity_type, $bundle = NULL) {
   /**
    * {@inheritdoc}
    */
-  public function entityUri($entity_type, $entity) {
-    $path = $entity->toUrl()->getInternalPath();
-    return $path;
+  public function entityUri($entity_type, $entity, $route = 'canonical') {
+    switch($route) {
+      case 'edit':
+      case 'delete':
+      case 'add':
+        $rel .= '-form';
+        break;
+      default:
+        return $entity->toUrl()->getInternalPath() . '/' . $route;
+    }
```

Comment:
In this case, if the method takes the default parameter 'canonical', the URL would be 'entity/[id]/canonical'.

Suggestion:
```suggestion
      default:
        $rel = $route;
    }
    return $entity->linkHasTemplate($rel) ? $entity->toUrl($rel)->getInternalPath() :  $entity->toUrl()->getInternalPath() . '/' . $route;
```

Reply:
It definitely makes more sense and covers the necessary cases. Thank you!